Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -226,6 +226,13 @@ public extension Node where Context == HTML.FormContext {
         .attribute(named: "action", value: url.string)
     }
     
+    /// Assign a target to the form, specifying where the response
+    /// received after submitting the form should be displayed.
+    /// - parameter target: The target to assign. See `HTMLAnchorTarget`.
+    static func target(_ target: HTMLAnchorTarget) -> Node {
```

Comment:
I think, along with this, we should rename `HTMLAnchorTarget` to something more appropriate (now that it's not only used for anchors) - for example `HTMLBrowserTarget`, since it's all about assigning a target for the browser to display a page in.